tinygrad/state.py

```diff
@@ -72,7 +72,7 @@ def _rebuild_tensor_v2(storage, storage_offset, size, stride, requires_grad, bac
     # upstream LLaMA also does this conversion:
     # https://github.com/facebookresearch/llama/blob/6c7fe276574e78057f917549435a2554000a876d/llama/generation.py#L95
     if storage[1] == dtypes.bfloat16:
-      ret = ret.to("LLVM").half().to(Device.DEFAULT).realize()
+      ret = ret.to("LLVM").half().to(Device.DEFAULT)
```

Comment:
Note @wozeparrot @prusnak - I removed this realize since it gets in the way of streaming the quanitzed data and spiking memory usage.

Reply:
should probably test with llama 2 to ensure that the cast is still happening, `.contiguous()` should enforce it a little bit better without requiring early realize.